dist/gmusic-theme.js

```diff
@@ -1008,155 +1013,155 @@ Converter.prototype.getValues = function(space) {
 
 module.exports = convert;
 },{"./conversions":3}],5:[function(require,module,exports){
-module.exports = {
```

Comment:
So how do we resolve this?  I guess we either accept that the format is going to be dependant on whatever platform the dev is using, unless there's a way of hard coding a format in a config somewhere?  Unfortunately i don't know enough about browserify (or babel, if that's the cause) to know if this is even an option, without digging into it a bit more...